README.md

```diff
@@ -19,10 +19,11 @@ The [Morphology Connectivity Hub](https://clue.io/morphology) is the primary sou
 We apply a unified, image-based profiling pipeline to all 136 384-well plates from `LINCS Pilot 1`, and all 135 384-well plates from `LKCP`.
 We use [pycytominer](https://github.com/cytomining/pycytominer) as the primary tool for image-based profiling.
 
-We process and store profiles in the [profiles/](profiles/) directory.
-See [`profiles/README.md`](profiles/README.md) for more details and for instructions on how to reproduce the pipeline.
+We process and store profiles in the [profiles/](profiles/) directory. By 'profiles' we refer to the level 3 data, which contains the aggregate of each well.
```

Comment:
We call everything from level 3-5 profiles. Also doesn't this folder contain level 4a and 4b profiles?